package.json

```diff
@@ -7,9 +7,9 @@
     "axios": "^0.18.0",
     "crypto": "^1.0.1",
     "dotenv": "^6.0.0",
-    "gatsby": "next",
```

Comment:
@alexiscott I'm not sure why we are removing "next" and specifying a min version? From my understanding "next" ensures we always have the latest version when we try to run an update.

Reply:
@iris-i  I think we might want to tag it to "latest" as otherwise it seems to be tagged to an older version:
See the 
https://www.npmjs.com/package/gatsby?activeTab=versions

Reply:
Same for https://www.npmjs.com/package/gatsby-plugin-react-helmet I believe.

Reply:
Looks good. Thanks for updating this @alexiscott!